modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -90,9 +92,11 @@ public ObjectAction addObjectAction(
 			Map<Locale, String> errorMessageMap, Map<Locale, String> labelMap,
 			String name, String objectActionExecutorKey,
 			String objectActionTriggerKey,
-			UnicodeProperties parametersUnicodeProperties)
+			UnicodeProperties parametersUnicodeProperties, boolean system)
 		throws PortalException {
 
+		_validateInvokerBundle("add", system);
```

Comment:
Refact this methos like [this one](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java#L90-L92)

Reply:
@gabrielwas makes sense. Should I use "add" or "create" in the message?

We are using "add" for [the Object Fields](https://github.com/liferay/liferay-portal/blob/54237fd517a688694860af0394d17fb0b6e3b501/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java#L291-L292).

Reply:
Same as validationRules. We can change in object fields latter